src/algebra/module/linear_map.lean

```diff
@@ -551,6 +555,8 @@ instance : has_sub (M →ₛₗ[σ₁₂] N₂) :=
           map_add' := λ x y, by simp only [pi.sub_apply, map_add, add_sub_comm],
           map_smul' := λ r x, by simp [pi.sub_apply, map_smul, smul_sub] }⟩
 
+lemma coe_sub (f g : M →ₛₗ[σ₁₂] N₂) : ⇑(f - g) = f - g := rfl
```

Comment:
This appears to have resulted in a CI failure.

```
2021-11-09T13:04:25.4326281Z ##[error]/home/lean/actions-runner/_work/mathlib/mathlib/src/analysis/convex/cone.lean:468:4:
ambiguous overload, possible interpretations
  submodule.coe_sub
  linear_map.coe_sub
```

Reply:
Replacing `coe_sub` with `submodule.coe_sub` on the affected line will fix it.

Reply:
One the one hand this was kind of out of scope, but on the other hand if we're adding lemmas this deep in the import graph it's nice to knock out all the similar ones at the same time. If you feel strongly you can back the `coe_add`, `coe_sub`, and `coe_zero` lemmas out.

Reply:
I've reverted that commit. It may make more sense to put those other lemmas in a separate PR (along with corresponding `multilinear_map` and `alternating_map` lemmas).